src/EditorFeatures/VisualBasicTest/Diagnostics/Suppression/SuppressionTests.vb

```diff
@@ -1463,7 +1483,7 @@ End Class]]>
 ' Project-level suppressions either have no target or are given
 ' a specific target and scoped to a namespace, type, member, etc.
 
-<Assembly: Diagnostics.CodeAnalysis.SuppressMessage("InfoDiagnostic", "InfoDiagnostic:InfoDiagnostic", Justification:="<Pending>", Scope:="type", Target:="Class1")>
+<Assembly: SuppressMessage("InfoDiagnostic", "InfoDiagnostic:InfoDiagnostic", Justification:="<Pending>", Scope:="type", Target:="Class1")>
```

Comment:
Should we fix the input GlobalSuppressions doc to use a fully qualified name here to avoid compiler errors in input?

Reply:
done.

Reply:
#resolved